src/library/Collections.nim

```diff
@@ -76,56 +76,55 @@ proc defineSymbols*() =
             # TODO(Collections\append) Append for arrays vs strings seems to be much slower
             #  https://github.com/arturo-lang/benchmarks/blob/main/results/17-8-2022/micro.md
             #  labels: enhancement, library, performance
-            if x.kind==Literal:
-                if InPlace.kind==String:
-                    if y.kind==String:
+            if x.kind == Literal:
+                if InPlace.kind == String:
+                    if y.kind == String:
                         InPlaced.s &= y.s
-                    elif y.kind==Char:
+                    elif y.kind == Char:
                         InPlaced.s &= $(y.c)
-                elif InPlaced.kind==Char:
-                    if y.kind==String:
+                elif InPlaced.kind == Char:
+                    if y.kind == String:
                         SetInPlace(newString($(InPlaced.c) & y.s))
-                    elif y.kind==Char:
+                    elif y.kind == Char:
                         SetInPlace(newString($(InPlaced.c) & $(y.c)))
                 else:
-                    if y.kind==Block:
+                    if y.kind == Block:
                         ensureCleaned(y)
```

Comment:
```suggestion
```

We don't really need this (to pre-clean the `y` value that is), since `cleanAppend` is supposed to work on non-clean values after all.

Reply:
I have a doubt. What exactly is a clean value?

Reply:
You answered my question in [this comment](https://github.com/arturo-lang/arturo/pull/755#issuecomment-1277306758). Thank you!

Now it sees clear to me.